.coveragerc

```diff
@@ -0,0 +1,5 @@
+[run]
+omit =
+    src/histogram/_version.py
+source =
```

Comment:
what about here?

Reply:
I can add bin and lib folder here but it wouldn't run
![image](https://github.com/user-attachments/assets/0b62bc24-a088-45da-a465-9a7c1beb7ca9)